python/cudf/cudf/_lib/pylibcudf/experimental.pyx

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+from libcpp cimport bool
+from libcpp.string cimport string
+
+from cudf._lib.pylibcudf.libcudf cimport experimental as cpp_experimental
+
+
+cpdef enable_prefetching(str key):
```

Comment:
Can you add docstrings?

Reply:
Good catch, thanks. I've spent most of the time on this PR staring at the C++.